### DISCUSSION TITLE
AM Band: None

### DISCUSSION THREAD

**Discussion by AaronJowitt** [PE5PVB/TEF6686_ESP32#419] [Ideas]
AM Settings > Select AM Band.  'None' option. When only using it for FM, it prevents inadvertent band changes.

**Comment by ohmytime** [comment]
What version are you on now?
I'm using the latest version of the code, switching AM bands is no problem

**Reply by AaronJowitt** [reply]
2.00 RC1.  

Yes, I can currently change AM bands 'LW,SW,MW' (and various combinations) fine.  I'm suggesting a new "none' option, so AM bands 'LW,SW,MW,None' 

Selecting 'none' disables all AM bands, effectively disabling the band button and making it an FM only radio. 

Prevents accidental changing to an AM band you never want.

**Comment by ohmytime** [comment]
> 2.0 RC1.
> 
> Yes, I can currently change AM bands 'LW,SW,MW' (and various combinations) fine. I'm suggesting a new "none' option, so AM bands 'LW,SW,MW,None'
> 
> Selecting 'none' disables all AM bands, effectively disabling the band button and making it an FM only radio.
> 
> Prevents accidental changing to an AM band you never want.

I understand now, I will look into it.

**Comment by ohmytime** [comment]
Done, please test #436

**Reply by AaronJowitt** [reply]
Done for AM & FM bands - good idea. Nice, thanks!  
Will test next discord update.

**Comment by AaronJowitt** [comment]
> Done, please test #436

Yes, it seems to work fine.  If you have only one valid band configured (i.e. just FM, or just MW) and press band change button, it reloads the screen and mutes audio for a fraction of a second. Not sure if intended, but it's minor.

It does exactly what it's meant to do. Useful - thank you!

**Reply by ohmytime** [reply]
Yes , I know that, I can completely disable this, but I think it's better to show a hint , that band button press gives user a feedback.
